dns/zone-configs/x-k8s.io._0_base.yaml

```diff
@@ -1,3 +1,4 @@
 # The top-level x-k8s.io zone
-
-# Nothing here.
+'':
+  type: CNAME
```

Comment:
I think that cnames are not allowed at root level, 
the octodns code seems to confirm that
https://github.com/octodns/octodns/blob/e087a3b26f0272a994d3efba2a62ead0f4617a16/octodns/record/__init__.py#L886

Reply:
the first approach with the A seems the simplest/safest, no?